contrib/arm/inflate.c

```diff
@@ -0,0 +1,1582 @@
+/* inflate.c -- zlib decompression
+ * Copyright (C) 1995-2016 Mark Adler
```

Comment:
I wonder if we should point clearly that this is a modded inflate.c (i.e. by adding the respective Copyright).